src/storage/src/hummock/local_version/pinned_version.rs

```diff
@@ -12,11 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::collections::BTreeMap;
+use std::collections::{BTreeMap, HashMap};
 use std::ops::Deref;
 use std::sync::Arc;
 use std::time::Duration;
 
+use risingwave_hummock_sdk::compaction_group::hummock_version_ext::HummockVersionExt;
+use risingwave_hummock_sdk::compaction_group::StateTableId;
```

Comment:
can we use `TableId` instead of `StateTableId` , we can reduce multiple variables that unify a concept

Reply:
> can we use `TableId` instead of `StateTableId` , we can reduce multiple variables that unify a concept

fixed